runtimes/catboost/mlserver_catboost/catboost.py

```diff
@@ -0,0 +1,35 @@
+from catboost import CatBoostClassifier
+
+from mlserver import types
+from mlserver.model import MLModel
+from mlserver.utils import get_model_uri
+from mlserver.codecs import NumpyCodec, NumpyRequestCodec
+
+
+WELLKNOWN_MODEL_FILENAMES = ["model.bst"]
```

Comment:
The "well-known" filenames list is used to try to automatically find the right artifacts when they are not specified explicitly in the `model-settings.json`. We can think of it as a list of "standardised" names. 

Based on the CatBoost example, should `model.cbm` be considered as a well known filename? The `model.bst` seems more relevant to XGBoost, but I'm not very familiar with CatBoost, so it would be great to hear your thoughts!

Reply:
nice catch! addressed in d49909e, searching github for how people name their models, bring the cbm and bin as the two most popular extensions for catboost